src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Shapes/Polyline.cs

```diff
@@ -2,25 +2,11 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.UI.Xaml.Shapes
 {
-	#if __ANDROID__ || __IOS__ || NET461 || false || __MACOS__
+	#if false || false || false || false || false
 	[global::Uno.NotImplemented]
 	#endif
-	public  partial class Polyline : global::Windows.UI.Xaml.Shapes.Shape
+	public  partial class Polyline
```

Comment:
It's not ideal to change the base type. How much effort would it be to extract the functionality you need from `ArbitraryShapeBase` to shared code?